### PR TITLE
[pubsub] New format for go-ipfs >= 0.11.0

### PR DESCRIPTION
**Pull request by odesenfans**
Modified the format used to subscribe and publish on pubsub topics.
go-ipfs v0.11.0 introduces a new format requiring multibase
base64url encoding for most parameters.
Additionally, published data must now be added as a multipart form
instead of a URL parameter.

### CODE REVIEW

**Comment by hoh** [comment]
Upstream on https://gitlab.com/cipres/aioipfs/-/issues/4 ?

File: aioipfs/api.py [hunk 1]
```diff
@@ -1004,16 +1009,29 @@ async def peers(self, topic=None):
...
         Publish a message to a given pubsub topic.
 
         :param str topic: topic to publish the message to
         :param str data: message data
         """
 
-        return await self.post(self.url('pubsub/pub'),
-                               None, params=quote_args(topic, data))
+        params = {
+            ARG_PARAM: multibase.encode(
```
Comment (hoh) [inline aioipfs/api.py:1021]: Why not using `base64.urlsafe_b64encode` ?

Comment (odesenfans) [reply aioipfs/api.py:1021]: Same encoding in the end, right? I went straight to the point: the new spec says multibase, I used the multibase package.

Comment (hoh) [reply aioipfs/api.py:1021]: I see.

**Review by hoh — APPROVED** [review]

**Pull request merged**